lua/better_escape.lua

```diff
@@ -72,7 +72,10 @@ local function record_key(key)
     end)
 end
 
-vim.on_key(function()
+vim.on_key(function(mapping, typed)
```

Comment:
```suggestion
vim.on_key(function(_, typed)
```
to get no `unused variable` warnings